Token.js

```diff
@@ -643,37 +671,13 @@ class Token {
 				old.find(".hpbar").show();
 				old.find(".ac").show();
 			}
-
-			let scale = (((this.options.size - 15) * 100) / this.options.size) / 100;
-			if (!(this.options.max_hp) > 0 || (this.options.disableaura))
-				scale = 1;
+			let scale = this.get_token_scale()
 			var rotation = 0;
 			if (this.options.rotation != undefined) {
 				rotation = this.options.rotation;
 			}
 			old.find("img").css("transform", "scale(" + scale + ") rotate("+rotation+"deg)");
-
```

Comment:
this block is now called via the 2 new functions at the end of place, so it will be applied to both new and old tokens